pom.xml

```diff
@@ -128,7 +128,7 @@
     <json-path.version>2.2.0</json-path.version>
     <json.version>20160212</json.version>
     <netty.version>4.1.16.Final</netty.version>
-    <netty-http.version>1.1.0-SNAPSHOT</netty-http.version>
```

Comment:
I did this already. Is this from restoring the branch from an earlier point?

Reply:
Yes. It was one commit ahead. 

![image](https://user-images.githubusercontent.com/1472867/46772732-54918980-ccae-11e8-877e-b94d053f9b14.png)